src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkContextAuction.tsx

```diff
@@ -98,6 +98,8 @@ export const ArtworkContextAuctionFragmentContainer = createFragmentContainer<{
         ...RelatedWorksArtworkGrid_artwork
       }
       sales(size: 4, sort: TIMELY_AT_NAME_ASC) {
+        ...OtherAuctions_sales
+        # Relay isn't transitively expanding the child fragments for some reason, so we'll expand manually.
         ...AuctionCard_sale
```

Comment:
I've read [the docs](https://facebook.github.io/relay/docs/en/graphql-in-relay.html) again but still can't figure out why this isn't working. I'm guessing it might be due to limitations on how complex the root query can be, but this same pattern is used elsewhere in Reaction without any problems. (This is also causing the `as any` casts in the above props.)

Any ideas, @alloy?

Reply:
The problem is that `<OtherAuctions sales={sales as any} />` lines above isn't importing the right component.

If you look at the import: https://github.com/artsy/reaction/pull/2282/files#diff-e8d6463d830fb8f6a30d2867e8da582cL9 , that's importing the vanilla React component , whereas you should be importing the Relay fragment container. That _should_ clear up the issues (`any` and essentially needing to do `@relay(mask: false)` in your fragments).

Reply:
Haha, I feel like every six months or so, I make this mistake 😅 Thanks Matt!